### ISSUE TITLE
Title: How to Launch the Graphical User Interface of Whisperer Project?

### ISSUE THREAD

**Issue by joy1997520** [tigros/Whisperer#27]
Content:
Hello, I have recently been trying to utilize the Whisperer project but encountered some difficulties. I have configured all the necessary dependencies as per the instructions in README.md and successfully cloned the project to my local machine. However, I am unclear on how to launch the Graphical User Interface (GUI). I attempted to double-click main.exe, but only saw a command prompt flash and then disappear. I also tried opening whisperer.sln through Visual Studio, but did not see the expected graphical interface. Are there specific steps or commands to launch the GUI? I greatly appreciate your assistance!

**Comment by tigros** [comment]
You see on the right "Releases" click that, then download latest Whisperer.zip. Extract zip to a new folder and run whisperer.exe

**Comment by joy1997520** [comment]
> You see on the right "Releases" click that, then download latest Whisperer.zip. Extract zip to a new folder and run whisperer.exe


Dear Developer
I cannot thank you enough for your timely and precise guidance. Before reaching out to you, I spent over 10 hours wrangling with this issue, nearly pulling an all-nighter yesterday. I sought help from advanced language models like ChatGPT-4 and Google Bard, but to no avail. Your response shines a light on the importance of human interaction in problem-solving. I followed your instructions, and now the Whisperer project is up and running smoothly on my machine. Your help has saved me from further agonizing hours of troubleshooting.

Warmest thanks,
[Joyoldschool]
